### PR TITLE
[Resource] Add Host

### PR DESCRIPTION
**Pull request by michaelkad**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
=== RUN   TestAccIBMIHostBasic
--- PASS: TestAccIBMIHostBasic (83.42s)
PASS
...
```

### CODE REVIEW

**Review by ismirlia — APPROVED** [review]

File: ibm/service/power/resource_ibm_pi_host.go [hunk 1]
```diff
@@ -0,0 +1,374 @@
...
+			},
+			Arg_HostGroupID: {
+				Description:  "ID of the host group to which the host should be added.",
+				ForceNew:     true,
+				Required:     true,
+				Type:         schema.TypeString,
+				ValidateFunc: validation.NoZeroValues,
+			},
+			Arg_Host: {
+				Description: "Host to add to a host group.",
```
Comment (hkantare) [inline ibm/service/power/resource_ibm_pi_host.go:54]: Same comment as other res pr

**Comment by hkantare** [comment]
resolve conflicts

File: ibm/service/power/resource_ibm_pi_host.go (hunk 1 above)
Comment (michaelkad) [reply ibm/service/power/resource_ibm_pi_host.go:54]: Updated.

**Comment by michaelkad** [comment]
> resolve conflicts

Resolved

**Review by hkantare — APPROVED** [review]

**Pull request merged**